Inc/BufferHelpers.h

```diff
@@ -91,7 +91,7 @@ namespace DirectX
         unsigned int bindFlags = D3D11_BIND_SHADER_RESOURCE) noexcept;
 
     // Strongly typed wrapper around a Direct3D constant buffer.
-    namespace Internal
+    namespace Private
```

Comment:
Had to change this namespace to avoid warnings with DirectXCollision.inl when I tried to add this to a inline namespace.